src/components/select/bl-select.css

```diff
@@ -0,0 +1,218 @@
+:host {
+  width: 200px;
+  display: inline-block;
+
+  --bl-select-menu-z-index: 1;
+}
+
+.select-wrapper {
+  width: 100%;
+  position: relative;
+
+  --bl-select-padding-vertical: var(--bl-size-2xs);
+  --bl-select-padding-horizontal: var(--bl-size-xs);
+  --bl-select-background-color: #fff;
+  --bl-select-border-color: var(--bl-color-border);
+  --bl-select-border-focus-color: var(--bl-color-primary-hover);
+  --bl-select-icon-color: var(--bl-color-content-tertiary);
+  --bl-select-text-color: var(--bl-color-content-primary);
+  --bl-select-label-color: var(--bl-color-content-secondary);
+  --bl-select-placeholder-color: var(--bl-color-content-tertiary);
+  --bl-select-height: var(--bl-size-2xl);
+  --bl-select-menu-padding: 0 var(--bl-size-m);
+  --bl-select-menu-margin-top: var(--bl-size-2xs);
+  --bl-select-font-size: var(--bl-font-size-m);
+  --bl-select-disabled-color: var(--bl-color-tertiary);
+  --bl-select-menu-height: 250px;
```

Comment:
No need to use `bl-select` prefixes in local variables. Since those variables can only be used inside this component, just use variables without prefixes. Like `--padding-vertical`